aca_preview/preview.py

```diff
@@ -650,31 +651,20 @@ def check_acq_p2(self):
         elif P2 < P2_lim + 1:
             self.add_message('warning', f'P2: {P2:.2f} less than {P2_lim + 1} for {obs_type}')
 
-    def check_guide_fraction(self):
+    def check_guide_count(self):
         """Check for sufficient guide star fractional count
 
         """
-        if self.is_ER and self.guide_count < 3.0:
+        if self.is_ER and self.guide_count_9th < 3.0:
             self.add_message(
                 'critical',
-                f'ER guide count: only {self.guide_count} bright stars for mag')
-        if self.is_OR and self.guide_count < 4.0:
+                f'ER guide count: only {self.guide_count_9th} bright stars for mag')
```

Comment:
How about the message `{obs_type} count of 9th mag guide stars {self.guide_count_9th:.2f} < 3.0`.  Then the corresponding message below is `{obs_type} count of guide stars {self.guide_count:.2f} < {count_lim}`

Reply:
Fixed.